packages/react-interactions/events/src/dom/testing-library/index.js

```diff
@@ -158,6 +158,33 @@ function testWithPointerType(message, testFn) {
   });
 }
 
+function emulateBrowserTab(backwards) {
```

Comment:
Please can you move this into the test file you're using it in, for now

Reply:
Sure. Will do in a follow up! :)